.github/workflows/cron-checks.yml

```diff
@@ -74,7 +74,7 @@ jobs:
     name: Test LLC (Debug)
     strategy:
       matrix:
-        xcode: [13.1, 13.4.1, 14.0.1]
+        xcode: [13.1, 13.4.1, 14.1]
```

Comment:
Do we actually still need 13.1? Isn't 13.4.1 enough?

Reply:
I think we still have to check the compatibility with Swift 5.5
<img width="512" alt="Screen Shot 2022-11-15 at 5 37 30 PM" src="https://user-images.githubusercontent.com/20794991/201988554-17c486b1-512a-4a33-84fa-0c7e103e6c25.png">